apps/web/app/future/availability/[schedule]/page.tsx

```diff
@@ -23,4 +29,36 @@ export const generateMetadata = async ({ params }: PageProps) => {
   );
 };
 
-export default WithLayout({ getLayout: null, Page: AvailabilitySettingsWebWrapper });
+const Page = async ({ params }: PageProps) => {
+  const h = headers();
+  const nonce = h.get("x-nonce") ?? undefined;
+  const session = await getServerSession(AUTH_OPTIONS);
+  const userId = session?.user?.id ?? -1;
+  const scheduleId = params?.schedule ? Number(params.schedule) : -1;
+  const [schedule, travelSchedules] = await Promise.all([
+    ScheduleRepository.findDetailedScheduleById({
+      scheduleId,
+      isManagedEventType: false,
+      userId,
+      timeZone: "UTC",
+      defaultScheduleId: -1,
```

Comment:
it seems like `session.user` doesn't include `timeZone` or `defaultScheduleId`. Where can I obtain these props? Or should we modify the `AUTH_OPTIONS` logic to acquire these props?